gateway/src/chains/ethereum/ethereum.ts

```diff
@@ -85,12 +85,13 @@ export class Ethereum extends EthereumBase implements Ethereumish {
   }
 
   public metricLogger(): void {
-    logger.info(
+    const metricFormat: string =
       this.requestCount +
-        ' request(s) sent in last ' +
-        this.metricsLogInterval / 1000 +
-        ' seconds.'
-    );
+      ' request(s) sent in last ' +
+      this.metricsLogInterval / 1000 +
+      ' seconds.';
+    logger.info(metricFormat);
+    logger.http(metricFormat);
```

Comment:
This feels like a hack to me. In Datadog, metrics are separated from logs - they are not the same entities. Do we have separate endpoints / functionalities in the reporting proxy API for logs and metrics?